includes/class-banana-crystal-woocommerce.php

```diff
@@ -138,6 +160,18 @@ public function init_form_fields() {
 				'type'		=> 'text',
 				'desc_tip'	=> __( 'This is your BananaCrystal store username.', 'wo-banana-crystal' ),
 
+			),
+			'subscriptions_enabled' => array(
+				'title'		=> __( 'Enable / Disable Subscriptions', 'wo-banana-crystal' ),
+				'label'		=> __( 'Enable subscriptions for this payment gateway', 'wo-banana-crystal' ),
+				'type'		=> 'checkbox',
+				'default'	=> 'no',
+			),
+			'subscription_key' => array(
+				'title'		=> __( 'Subscription Key', 'wo-banana-crystal' ),
+				'type'		=> 'text',
+				'desc_tip'	=> __( 'This is your BananaCrystal subscription key.', 'wo-banana-crystal' ),
```

Comment:
@armansheikh is it possible to hide/obfuscate this subscription key so once saved an admin can't get the full key (e.g. ***********94bea  and also encrypt it the db